packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -129,6 +131,19 @@ const useMenuItems = (): NavItemProps[] => {
         icon: <AppIcon />,
         href: '/account/integrations',
       },
+      { label: 'Bookmarks', isHeader: true },
+      { label: 'Bookmarks', icon: <BookmarkIcon />, href: '/bookmarks' },
+      {
+        label: 'New folder',
+        icon: <PlusIcon />,
+        onClick: () =>
+          openModal({
+            type: LazyModal.BookmarkFolder,
+            props: {
+              onSubmit: () => {},
+            },
+          }),
+      },
```

Comment:
This is not the right place right?
It should live in the sidebar, don't think you can do it from profile menu (at least I don't see the designs for this?)

Reply:
Yes for now we don't have a place that fire this modal. 
We could add it to Storybook in order to test (since is a dummy component) 👀

Reply:
@rebelchris It's like Luca said, the modal is in the design, but not how to activate it. Was gonna comment that, but forgot.

Put it here cause it made sense, but not sure if it's the right place to put it. Can ask design

Reply:
We should show it on sidebar but sidebar changes are currently on WIP

Reply:
Yeah it's from the sidebar indeed :) 
Please let's not add this to avoid it accidently ending up in prod.

You can commit without it invoked and we can link once sidebar is there?